chef_master/source/custom_resources.rst

```diff
@@ -21,14 +16,18 @@ For example, Chef includes built-in resources to manage files, packages, templat
 
 .. end_tag
 
+Custom resources were introduced in Chef version 12.5 and are now the preferred method of writing your own resources in Chef. If you are using an older version of the chef-client, please use the version picker (in the top left of the navigation) to select your version, and then choose the same topic from the navigation tree ("Extend Chef > Custom Resources"). See also https://github.com/chef-cookbooks/compat_resource for using custom resources with chef-client 12.1 - 12.4.
```

Comment:
We no longer have the version picker in the main doc set; however, you can refer people off to the archive site for the older docs. The "Custom Resources" topic for the Chef 12.4 doc set is here: https://docs-archive.chef.io/release/12-4/custom_resources.html